CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set(CPACK_PACKAGE_VERSION_PATCH "0")
 set(COMPLETE_VERSION ${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH})
 set(RELEASE_NAME "Master")
 project(qgis VERSION ${COMPLETE_VERSION})
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
I'm using LSP, and it's better to add this option at command line so it doesn't bother other people building QGIS without LSP